monitor/tasks.py

```
@@ -0,0 +1,489 @@
+        features_array = np.array(features)
+        
+        # Detect anomalies
+        isolation_forest = IsolationForest(
+            contamination=0.1,  # Expect 10% anomalies
+            random_state=42
+        )
+        anomaly_labels = isolation_forest.fit_predict(features_array)
+        anomaly_scores = isolation_forest.score_samples(features_array)
+        
+        anomalies = []
+        for i, (label, score) in enumerate(zip(anomaly_labels, anomaly_scores)):
```

Ruff (0.12.2)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)